velox/functions/prestosql/aggregates/SumAggregate.h

```diff
@@ -148,7 +148,9 @@ class SumAggregate
   template <typename TData>
   static void updateSingleValue(TData& result, TData value) {
     if constexpr (
-        std::is_same_v<TData, double> || std::is_same_v<TData, float>) {
+        std::is_same_v<TData, double> || std::is_same_v<TData, float> ||
+        std::is_same_v<TData, int8_t> || std::is_same_v<TData, int16_t> ||
+        std::is_same_v<TData, int32_t> || std::is_same_v<TData, int64_t>) {
       result += value;
```

Comment:
What is the type of reuslt? Maybe accumulated type should be used for the result here.

Reply:
Looks like the accumalted types for int8/int16/int32 are all int64. Maybe we only need to handle int64 here.
```
          case TypeKind::TINYINT:
            return std::make_unique<T<int8_t, int64_t, int64_t>>(BIGINT());
          case TypeKind::SMALLINT:
            return std::make_unique<T<int16_t, int64_t, int64_t>>(BIGINT());
          case TypeKind::INTEGER:
            return std::make_unique<T<int32_t, int64_t, int64_t>>(BIGINT());
          case TypeKind::BIGINT:
            return std::make_unique<T<int64_t, int64_t, int64_t>>(BIGINT());
```

Reply:
Fixed